### PR TITLE
[docs] Delete superfluous bundles and update bundle docs

### PR DESCRIPTION
**Pull request by SplitInfinity**
**Description**
This commit removes the zfnet512 and vgg19 bundles because they are not
really necessary if there are two (resnet50 and lenet) working examples
of how to create standalone bundles. There are also some minor cmake
improvements.

**Testing**
ResNet50 bundles successfully build.

**Documentation**
This commit updates `docs/AOT.md` so that it explains the new `cmake` flow
for building standalone bundles.

**Issue**
This commit is part of #1979.

### CODE REVIEW

File: docs/AOT.md [hunk 1]
```diff
@@ -175,19 +171,18 @@ The makefile provides the following targets:
...
-  into a standalone executable file called `resnet50_standalone`
-  * `run`: it runs this standalone executable with imagenet images as inputs and outputs the
-results of the network model execution.
+ * `ResNet50Bundle`: it links the user-defined `main.o` and auto-generated `resnet50.o` into a standalone executable file called `resnet50`
+  * `RunResNet50Bundle`: it runs this standalone executable with imagenet images as inputs and outputs the results of the network model execution.
 
 ### Quantized network
-To build and run the example, you just need to execute:
-* `make run`. By default, quantized Resnet50 example is going to be executed.
+All of the aforementioned targets have quantized versions in CMakeLists.txt named `...QuantizedResNet50...`. For example, to run the quantized bundle, you just need to execute:
```
Comment (bertmaher) [inline docs/AOT.md]: IMO the "named `...QuantizedResNet50...`" bit is a little confusing.  I think the example `ninja` command below is clear enough.

File: docs/AOT.md [hunk 2]
```diff
@@ -148,25 +148,21 @@ variables area.
...
 
-There are concrete examples of integrating a network model with a project. You
-can find them in the `examples/bundles/` directory in the Glow repository.
-Makefiles with appropriate targets are provided for your convenience.
+There are concrete examples of integrating a network model with a project located in the `examples/bundles/` directory in the Glow repository. You can enable the compilation of these bundles by invoking `cmake` with `-DGLOW_WITH_BUNDLES=ON`.
 
 ### Floating point network
 To build and run the example, you just need to execute:
-* `QUANTIZE=NO make run`
+* `cmake ... -DGLOW_WITH_BUNDLES=ON ...`
```
Comment (bertmaher) [inline docs/AOT.md]: The `...` is potentially confusing, I'd phrase it something like:
```
cmake -DGLOW_WITH_BUNDLES=ON  # ... other cmake flags ...
```

**Comment by SplitInfinity** [comment]
- Updated references to `make` and Makefile in Overview section
- Addressed comments about unclear documentation

File: docs/AOT.md [hunk 3]
```diff
@@ -9,20 +9,18 @@ time.
...
 A bundle is a self-contained compiled network model that can be used to execute
 the model in a standalone mode. After following the instructions in this
-document and the [Makefile](../examples/bundles/resnet50) in the example
+document and the [CMakeLists.txt](../examples/bundles/resnet50/CMakeLists.txt) in the example
 directory you will be able to compile convolutional neural networks into small
 executables. Example:
 
 ```
-  $make
+  $cmake <other cmake flags> -DGLOW_WITH_BUNDLES=ON
```
Comment (rdzhabarov) [inline docs/AOT.md]: can you add GLOW_WITH_CPU as well, since otherwise it does not work

Comment (rdzhabarov) [reply docs/AOT.md]: Or mention in the doc that compiling with CPU backend is required here.

File: docs/AOT.md [hunk 4]
```diff
@@ -148,25 +146,21 @@ variables area.
...
+There are concrete examples of integrating a network model with a project located in the `examples/bundles/` directory in the Glow repository. You can enable the compilation of these bundles by invoking `cmake` with `-DGLOW_WITH_BUNDLES=ON`.
 
 ### Floating point network
 To build and run the example, you just need to execute:
-* `QUANTIZE=NO make run`
-
-You may need to adjust the environment variables at the top to match
-your setup, primarily LOADER and GLOW_SRC vars.
+* `cmake <other cmake flags> -DGLOW_WITH_BUNDLES=ON `
+* `ninja RunResNet50Bundle`
```
Comment (rdzhabarov) [inline docs/AOT.md]: this only works if you use `Ninja` generator in the cmake command.

**Comment by SplitInfinity** [comment]
Rebased and addressed @rdzhabarov 's comments:
- Added `-DGLOW_WITH_CPU=ON1` flag to all `cmake` commands in `AOT.md` because this is necessary for bundle compilation
- Added `-G ninja` flag to all `cmake` commands so that the commands that use `ninja` to run executables make sense

**Pull request merged**